docs/source/10min_dask_cudf.ipynb

```diff
@@ -0,0 +1,2565 @@
+      "2.5 1.666666666666666\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(s.mean(), s.var())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 33,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "2.5 1.6666666666666667\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(ds.mean().compute(), ds.var().compute())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Applymap"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Applying functions to a `Series`. Not that applying user defined functions with dask_cudf is not yet implemented."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 34,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0    10\n",
+      "1    11\n",
+      "2    12\n",
+      "3    13\n",
+      "4    14\n",
+      "5    15\n",
+      "6    16\n",
+      "7    17\n",
+      "8    18\n",
+      "9    19\n",
+      "[10 more rows]\n",
+      "Name: a, dtype: int64\n"
+     ]
+    }
+   ],
+   "source": [
+    "def add_ten(num):\n",
+    "    return num + 10\n",
+    "\n",
+    "print(df['a'].applymap(add_ten))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Histogramming"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Counting the number of occurrences of each unique value of variable."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 35,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0    1\n",
+      "1    1\n",
+      "2    1\n",
+      "3    1\n",
+      "4    1\n",
+      "5    1\n",
+      "6    1\n",
+      "7    1\n",
+      "8    1\n",
+      "9    1\n",
+      "[10 more rows]\n",
+      "dtype: int64\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(df.a.value_counts())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 36,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0    1\n",
+      "1    1\n",
+      "2    1\n",
+      "3    1\n",
+      "4    1\n",
+      "5    1\n",
+      "6    1\n",
+      "7    1\n",
+      "8    1\n",
+      "9    1\n",
+      "[10 more rows]\n",
+      "dtype: int64\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(ddf.a.value_counts().compute())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## String Methods"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Like pandas, cuDF provides string processing methods in the `str` attribute of `Series`. Full documentation of string methods is a work in progress. Please see the cuDF and nvstrings API documentation for more information."
```

Comment:
My 2c but I don't want users using nvstrings directly as we have plans to refactor it out. cuDF should be the entrypoint for working with string data.
```suggestion
    "Like pandas, cuDF provides string processing methods in the `str` attribute of `Series`. Full documentation of string methods is a work in progress. Please see the cuDF API documentation for more information."
```